apps/ftu-ui/src/app/app.component.ts

```diff
@@ -1,11 +1,19 @@
-import { AfterContentInit, Component, HostBinding, inject } from '@angular/core';
+import { AfterContentInit, Component, HostBinding, Input, Output, inject } from '@angular/core';
 import { MatDialog, MatDialogConfig, MatDialogModule } from '@angular/material/dialog';
-import { selectQuerySnapshot } from '@hra-ui/cdk/injectors';
-import { StorageId, StorageSelectors } from '@hra-ui/cdk/state';
+import { dispatch, select$, selectQuerySnapshot } from '@hra-ui/cdk/injectors';
+import {
+  LinkRegistryActions,
+  ResourceRegistryActions,
+  StorageId,
+  StorageSelectors,
+  createLinkId,
+} from '@hra-ui/cdk/state';
 import { ScreenNoticeBehaviorComponent } from '@hra-ui/components/behavioral';
+import { ActiveFtuSelectors, IllustratorSelectors } from '@hra-ui/state';
 
 @Component({
-  selector: 'ftu-ui-root',
+  // eslint-disable-next-line @angular-eslint/component-selector
+  selector: 'hra-ftu-wc',
```

Comment:
Undo the selector change. It won't affect the web component as it is still registered under the new name in the module